clang/test/CodeGenHLSL/builtins/ddx-coarse.hlsl

```diff
@@ -0,0 +1,61 @@
+// RUN: %clang_cc1 -finclude-default-header  -x hlsl  -triple dxil-pc-shadermodel6.3-library %s \
+// RUN:  -emit-llvm -disable-llvm-passes -fnative-half-type -o - | \
```

Comment:
once you update the code add a second run line to  test the spirv target you should see the intrinsic change to `llvm.spv.ddx.coarse`